static/app/components/sidebar/index.spec.tsx

```diff
@@ -610,8 +591,15 @@ describe('Sidebar', function () {
 
       // The dot is not visible initially - banner takes precedence
       expect(screen.queryByTestId('help-menu-dot')).not.toBeInTheDocument();
-      expect(await screen.findByText(/Sentry has a new look/)).toBeInTheDocument();
-      await userEvent.click(screen.getByRole('button', {name: /Dismiss/}));
+      const chonkBanner = await screen.findByText(/Sentry has a new look/);
+      expect(chonkBanner).toBeInTheDocument();
+
+      // Find the dismiss button within the chonk UI banner
+      const bannerContainer =
+        chonkBanner.closest('[class*="Panel"]') || chonkBanner.parentElement;
```

Comment:
These kinds of selectors in tests are quite brittle so it's best to query by role. If the banner doesn't have a role, the next best thing would be to add a `data-test-id` on it and do `screen.getByTestId('...')`